manifests/1.2.3/opensearch-1.2.3.yml

```diff
@@ -68,6 +77,12 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: sql
```

Comment:
Rebase, this is already in the manifest.

Reply:
It is not @dblock https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.2.3/opensearch-1.2.3.yml

Reply:
Nevermind, it's me.